build/tutorials/tutorials-contest/NFT-Marketplace-on-Avalanche/.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules
```

Comment:
No need to have a .gitignore since we won't be having people install the apps in the same directory as the docs.